android/src/main/java/ch/milosz/reactnative/RNZoomUsInMeetingServiceListener.java

```diff
@@ -0,0 +1,297 @@
+package ch.milosz.reactnative;
```

Comment:
listener class for in meeting event.
commented code here is code for v5.2.4. I upgraded the sdk but then initialized method didn't work, so I downgrade it (but left the code, commented)

Reply:
Ok, but I really do not think it makes sense to keep commented code.
Could you at least make a `TODO` comnent (in the code)?